catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/MetacardApplication.java

```diff
@@ -474,6 +474,21 @@ public void init() {
         },
         util::getJson);
 
+    post(
+        "/unsubscribe/:id/:email",
+        (req, res) -> {
+          String email = req.params(":email");
+          String id = req.params(":id");
+          if (isEmpty(email)) {
+            throw new NotFoundException(
+                "Unable to un-subscribe from workspace, " + email + " is not a valid email.");
+          }
+          subscriptions.removeEmail(id, email);
```

Comment:
:nit: it's weird that removeEmail doesn't have any return value... what if the email wasn't subscribed?

Reply:
maybe it throws an exception...